src/Symfony/Component/Serializer/Encoder/XmlEncoder.php

```diff
@@ -173,7 +212,7 @@ public function supportsDecoding($format)
      */
     public function setRootNodeName($name)
```

Comment:
could be deprecated also, alongside with getRootNodeName?

Reply:
I don't why they have been introduced in the first time, but yes.